### DISCUSSION TITLE
What is the secret to using the String datatype for a metric?

### DISCUSSION THREAD

**Discussion by trentchristopher** [thomassorensen2650/node-red-contrib-mqtt-sparkplug-plus#59] [Q&A] [answered]
The UI doesn't offer a string datatype in the drop down, and when I pass in the definition, I get a "...invalid datatype String" message. The Sparkplug spec accommodates the string datatype. Is it just missing from this implementation?

**Comment by thomassorensen2650** [comment]
Ohh shoot.. The string datatype was removed by a mistake in the latest version that was released yesterday. 

I will see if I can release a new version today with the string datatype added back in.

**Comment by thomassorensen2650** [comment, marked as answer]
Please update to the newest version (V2.1.5). This version will have the string datatype again.

**Comment by trentchristopher** [comment]
You @thomassorensen2650 are a hero! Great work!
